src/backend/distributed/commands/cascade_table_operation_for_connected_relations.c

```diff
@@ -513,20 +514,28 @@ ExecuteCascadeOperationForRelationIdList(List *relationIdList,
 
 
 /*
- * ExecuteAndLogUtilityCommandListInTableTypeConversion is a wrapper function
- * around ExecuteAndLogUtilityCommandList, that makes it execute with the flag
- * InTableTypeConversionFunctionCall is set to true.
+ * ExecuteViewCreationCommandsViaSPI is a wrapper function
+ * around ExecuteQueryViaSPI, that executes view creation commands
+ * with the flag InTableTypeConversionFunctionCall set to true.
  */
 void
-ExecuteAndLogUtilityCommandListInTableTypeConversion(List *utilityCommandList)
+ExecuteViewCreationCommandsViaSPI(List *utilityCommandList)
```

Comment:
I think we can keep the name as ExecuteAndLogUtilityCommandListInTableTypeConversionViaSPI